vsphere/vmcdns.py

```diff
@@ -86,11 +87,17 @@ def change_json(action, route, base_name, ip):
 
 
 def debug(*a, **k):
-    if ARGS.debug:
-        print(*a, file=sys.stderr, **k)
-
+    # FIXME how to get ARGS from other script?
```

Comment:
Would be pretty hacky to import *all* the ARGS from the other script.

You could explicitly set `vmcdns.ARGS` up from the other script. Something like:
```python
vmcdns.ARGS = type('', (), dict(debug=ARGS.debug))
```
Or I suppose if you did want to import all the args, you could just do
```python
vmcdns.ARGS = ARGS
```